### ISSUE TITLE
Wildcard character (*) to @include multiple conf files doesn't work for absolute paths

### ISSUE THREAD

**Issue by gracedo** [fluent/fluent-bit#1294]
## Bug Report

Wildcards in the @INCLUDE config command only work for paths relative to the path of the base config file. Using absolute paths with wildcards causes Fluent Bit to fail to start up with configuration file errors. This should be a bug, because absolute paths without wildcards work fine.

**To Reproduce**
- Example log message:
```
systemd[1]: Started Fluent Bit: Collects and forwards logs.
fluent-bit[6931]: Fluent Bit v1.0.4
fluent-bit[6931]: Copyright (C) Treasure Data
fluent-bit[6931]: [2019/04/22 23:12:31] [Warning] [mk_rconf_read_glob] glob: no match
fluent-bit[6931]: Error: Configuration file contain errors. Aborting
```
- Steps to reproduce the problem:

Create base config file `main.conf`
```
@INCLUDE /tmp/fluent-bit/input_*.conf
@INCLUDE /tmp/fluent-bit/output_stdout.conf

[INPUT]
    Name    cpu
    Tag  cpu.local
    Interval_Sec 1
```

Create` /tmp/fluent-bit/input_mem.conf`
```
[INPUT]
    Name mem
    Tag  mem.local
```

Create `/tmp/fluent-bit/output_stdout.conf`
```
[OUTPUT]
    Name stdout
    Match *
```

Run Fluent Bit pointing to `main.conf` as the config file.
```
bin/fluent-bit -c main.conf
```

**Expected behavior**
Fluent Bit should successfully start up and run the cpu input plugin along with the mem input plugin and the stdout plugin. Instead, it fails to start up with the error logs mentioned above. Omitting the wildcard include line `@INCLUDE /tmp/fluent-bit/input_*.conf`, however, results in a successful Fluent Bit start up running the mem input plugin and the stdout output plugin.

**Comment by tinproject** [comment]
This at least should be documented, there is no clue in the configuration commands documentation about this behavior.

**Comment by elchead** [comment]
This issue still persists.

Relative paths using `..` also don't work, but relative paths in a subdirectory of the main config work.

**Comment by chakratn** [comment]
Can some one please let me know in which version this fix will be available? on 1.8.4, wildcard* issue is still seen.
